db/post_migrate/20210420162817_mark_badges_for_beginners.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class MarkBadgesForBeginners < ActiveRecord::Migration[6.0]
+  def up
+    execute <<~SQL
+      UPDATE badges
+      SET for_beginners = true
+      WHERE name IN (
+        'Autobiographer',
```

Comment:
You should be using the ids [from here](https://github.com/discourse/discourse/blob/f33b7c8f657eb348785402e1522b2e04f59ca881/app/models/badge.rb#L12-L72) otherwise it probably won't work if they changed the name of the badge or if they're using a different locale 😉